Shared/Samples/Display points using clustering feature reduction/DisplayPointsUsingClusteringFeatureReductionView.swift

```diff
@@ -60,28 +60,30 @@ struct DisplayPointsUsingClusteringFeatureReductionView: View {
                 .task(id: identifyScreenPoint) {
                     guard let identifyScreenPoint,
                           let layer,
-                          let identifyResult = try? await proxy.identify(
-                            on: layer,
-                            screenPoint: identifyScreenPoint,
-                            tolerance: 3
-                          ),
-                          let firstPopup = identifyResult.popups.first
-                    else { return }
-                    self.popup = firstPopup
-                    self.showsPopup = true
+                          let identifyResult = await Result(awaiting: {
+                              try await proxy.identify(
+                                on: layer,
+                                screenPoint: identifyScreenPoint,
+                                tolerance: 3
+                              )
+                          })
+                        .cancellationToNil()
+                    else {
+                        return
+                    }
+                    
+                    self.identifyScreenPoint = nil
```

Comment:
Why explicitly reset the identify point?

Reply:
This is also done in the Popup's example code. It's not 100% necessary so I'm happy to remove it if you prefer.

Reply:
No need to change.

Reply:
Final explanation is [above](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/247#discussion_r1297741574).